pkgs/applications/editors/emacs/elisp-packages/manual-packages.nix

```diff
@@ -45,6 +45,8 @@ in
 
   helm-words = callPackage ./manual-packages/helm-words { };
```

Comment:
We should replicate the by-name trick here too

Reply:
https://github.com/NixOS/nixpkgs/pull/330632